charts/argo-cd/Chart.yaml

```diff
@@ -22,3 +22,7 @@ dependencies:
     version: 0.0.2
     repository: https://charts.akuity.io
     condition: extensions.enabled
+  - name: argocd-image-updater
+    version: 0.0.11
```

Comment:
This was an old version and we need to update this when the new version of image updater is available for download. Otherwise, we will get an error:
```
Update Complete. ⎈Happy Helming!⎈
Error: can't get a valid version for repositories argocd-image-updater. Try changing the version constraint in Chart.yaml
```